src/Sylius/Behat/Context/Ui/Shop/AccountContext.php

```diff
@@ -73,13 +88,17 @@ public function __construct(
         ChangePasswordPageInterface $changePasswordPage,
         IndexPageInterface $orderIndexPage,
         ShowPageInterface $orderShowPage,
+        AddressBookIndexPageInterface $addressBookIndexPage,
+        AddressBookCreatePageInterface $addressBookCreatePage,
```

Comment:
Shouldn't we create separate Context for managing addresses instead of making this one even bigger?